extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -578,7 +578,9 @@ private void createRealm(String keycloakUrl, RealmRepresentation realm) {
                     .retry()
                     .withBackOff(Duration.ofSeconds(2), Duration.ofSeconds(2))
                     .expireIn(10 * 1000)
-                    .onFailure().transform(t -> t.getCause());
+                    .onFailure().transform(t -> {
+                        return new RuntimeException("Keycloak server is not available");
```

Comment:
Maybe we want to display the entire error here?

Reply:
Hi @geoand, with discussed a similar fix done by @famod for `OidcClientImpl`, the whole message can leak the internal connection details so we agreed to collapse it to make it informative enough, but for `DevServices` it can be more useful indeed, let me update it

Reply:
But this is code only run in dev mode, right?

Reply:
Yeah, sure, this is what I implied too that here it is of no real concern, so I'm just about to add an entire error message if it is available

Reply:
Done